pkgs/servers/monitoring/plugins/esxi.nix

```diff
@@ -1,18 +1,18 @@
-{ stdenv, fetchFromGitHub, python2Packages }:
+{ stdenv, fetchFromGitHub, python3Packages }:
 
 let
   bName = "check_esxi_hardware";
   pName = stdenv.lib.replaceStrings [ "_" ] [ "-" ] bName;
 
-in python2Packages.buildPythonApplication rec {
+in python3Packages.buildPythonApplication rec {
```

Comment:
There is an executable that is installed that has an extension, please remove the extension. Executables in `/bin` should not have any.